src/abstractdataframe/selection.jl

```diff
@@ -150,12 +152,15 @@ function select_transform!(nc::Pair{<:Union{Int, AbstractVector{Int}},
 end
 
 """
-    select!(df::DataFrame, inds...)
+    select!(df::DataFrame, args...)
 
-Mutate `df` in place to retain only columns specified by `inds...` and return it.
+Mutate `df` in place to retain only columns specified by `args...` and return it.
 
-Arguments passed as `inds...` can be any index that is allowed for column indexing.
-In particular, regular expressions, `All`, `Between`, and `Not` selectors are supported.
+Arguments passed as `args...` can be:
+
+* Any index that is allowed for column indexing. In particular, regular expressions,
+  `All`, `Between`, and `Not` selectors are supported.
+* Column transformation operations using the `Pair` notation that is described below.
 
```

Comment:
Also vectors of indices or pairs are allowed.

Reply:
I will add it